ui/tests/integration/components/secret-list-header-test.js

```diff
@@ -37,4 +39,63 @@ module('Integration | Component | secret-list-header', function (hooks) {
       }
     }
   });
+
```

Comment:
I wonder if we should add another test to make sure the modal opens and closes? If you decide to add it, make sure to check that it's active (doesn't just exist, since it's always in the DOM) - I've _maybe_ done that in a test before 😂